frontend/src/component/project/Project/hooks/useProjectForm.ts

```diff
@@ -172,6 +175,11 @@ const useProjectForm = (
             return false;
         }
 
+        if (projects.some(({ name }) => name === projectName)) {
```

Comment:
Will this also trigger on re-using the existing name on edit? E.g. if you begin editing project "A", and decide against it, but instead of cancelling, you input "A" and try to save: will it tell you that the name is already taken?